CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing to Roslyn
+
+Guidelines for contributing to the Roslyn repo.
+
+## Submitting Pull Requests
+
+For now, the team has set the following limits on pull requests:
+
+- Contributions beyond the level of a bug fix must be discussed with the team first, or they will likely be declined.
+- Pull requests must pass all Jenkins legs and be merge conflict free before they will be merged. 
+- Submissions must meet functional and performance expectations, including scenarios for which the team doesn’t yet have open source tests. This means you may be asked to fix and resubmit your pull request against a new open test case if it fails one of these tests.
+- Submissions must follow the [.editorconfig](http://editorconfig.org/) settings for each directory. 
+- Contributors must sign the [.NET CLA](https://cla2.dotnetfoundation.org/)
+
+When you are ready to proceed with making a change, get set up to [build](https://github.com/dotnet/roslyn/blob/master/docs/contributing/Building%2C%20Debugging%2C%20and%20Testing%20on%20Windows.md) the code and familiarize yourself with our developer workflow. 
```

Comment:
No idea.  I've tried using relative links and not had a lot of success.  Perhaps @sharwell knows a way. 

I agree it would be nice.  I just don't know if there is a way to do this.